test/types/component-test.ts

```diff
@@ -1,37 +1,66 @@
 import { expectTypeOf } from 'expect-type';
+
+// Intentionally checking the shape of the exports *and* the export itself.
 import * as gc from '@glimmer/component';
+// eslint-disable-next-line no-duplicate-imports
+import Component from '@glimmer/component';
```

Comment:
Linting doesn't seem to like this

Reply:
Ahhh, the problem is that it's tslint instead of eslint. Ugh.